features/attached_commands.feature

```diff
@@ -364,3 +364,65 @@ Feature: Command behaviour when attached to an UA subscription
            | focal   |
            | trusty  |
            | xenial  |
+
+    @series.trusty
+    Scenario: Enable command with invalid repositories in user machine
+        Given a `trusty` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token` with sudo
+        And I run `ua disable esm-infra` with sudo
+        And I run `add-apt-repository ppa:ua-client/proposed -y` with sudo
+        And I run `apt update` with sudo
+        And I run `sed -i 's/ubuntu/ubun/' /etc/apt/sources.list.d/ua-client-proposed-trusty.list` with sudo
+        And I run `ua enable esm-infra` with sudo
+        Then stdout matches regexp:
+        """
+        One moment, checking your subscription first
+        Updating package lists
+        APT update failed.
+        Failed to fetch http://ppa.launchpad.net/ua-client/proposed/ubun/dists/trusty/main/binary-amd64/Packages  404  Not Found .*
```

Comment:
I think we can cleanup an minimize the message a bit with a call to action for the person using UA CLI.

What do folks think about this mesaging

Unable to `ua enable esm-infra` due to APT update failure.
Fix apt config for http://ppa.launchpad.net/ua-client/proposed/ubun/dists/trusty/main/binary-amd64/Packages and retry.

Reply:
I see no problem on using that message instead of the full pat update error. What do you think @mitechie ?

Reply:
@blackboxsw What about updating the message to:

```
 Fix APT config for this repo and retry:
- http://ppa.launchpad.net/ua-client/proposed/ubun/dists/trusty/main/binary-amd64/Packages 
```

The rationale for that is situations were the user has more than one invalid ppas int the system

Reply:
Message looks pretty good, and I think a list of errant apt repos makes sense, but I'd defer to @mitechie on whether that list might get too long if we list 3 or more failing apt repo URLs.

Can the message be:
```
APT update failed to read APT config for the following URLs:
- http://ppa.launchpad.net/ua-client/proposed/ubun/dists/trusty/main/binary-amd64/Packages
- URL2
Fix any APT config for the URLs above and retry `ua enable <blah>`
```

Reply:
I am ok with noting the details of the error and if there are several that's ok. I do think that we need that summary message of "Unable to enable esm-infra due to APT update failures. Please address and retry"

Presenting the user the important details of the error to understand what's wrong is great. We should also always try to go the next step and give them the "what do I do next" as well.